docs/api.rst

```diff
@@ -1,7 +1,5 @@
 API
 ===
 
-.. autosummary::
-   :toctree: generated
-
-   europi
\ No newline at end of file
+.. automodule:: europi
+   :inherited-members:
```

Comment:
I think that you want to leave this file as is, and add the `:inherited-members:` config to `_templates/autosummary/module.rst`. Both of these changes will make it easier to add additional files in the future.

Reply:
Thanks for the tip, that worked great!

https://awonak.github.io/EuroPi/generated/europi.html#module-europi